src/main/java/se/uu/elephas/controllers/OrderController.java

```diff
@@ -0,0 +1,80 @@
+package se.uu.elephas.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import se.uu.elephas.model.Order;
+import se.uu.elephas.services.OrderServiceImpl;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/order")
+@CrossOrigin("*")
+public class OrderController {
+
+    @Autowired
+    private OrderServiceImpl orderService;
+
+    @RequestMapping(value = "/create", method = {RequestMethod.POST})
+    public ResponseEntity<String> create(
+            @RequestParam("userId") @Valid Long userId)
+            throws JsonProcessingException {
+
+
+        Order order = (Order) orderService.create(userId);
+        if (order == null)
+       // } catch (ConstraintViolationException e) {
+            return ResponseEntity.status(HttpStatus.OK).body("Cannot create order.");
```

Comment:
I believe that it's fine returning OK bc technically it's not a connection etc error